### PR TITLE
[CPU][ARM] Fix indices in ACL Interpolate executor

### PR DESCRIPTION
**Pull request by alvoron**
### Details:
 - Because of incorrect height and weight indices `is_upsample` variable is initialised incorrectly and NHWC is not selected, that leads to additional reorders in `octave-resnext`.

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp [hunk 1]
```diff
@@ -27,13 +15,7 @@ bool ov::intel_cpu::ACLInterpolateExecutor::init(const InterpolateAttrs &interpo
...
     auto& out_shape = dstDescs[0]->getShape().getDims();
 
-    int index_h, index_w;
-    if (!getIndices(dstDescs[0], index_h, index_w)) {
-        DEBUG_LOG("ACL Interpolate unsupported layout: ", dstDescs[0]->serializeFormat());
-        return false;
-    }
-
-    if ((aclInterpolateAttrs.coordTransMode == InterpolateCoordTransMode::pytorch_half_pixel && out_shape[index_h] > 1 && out_shape[index_w] > 1) ||
+    if ((aclInterpolateAttrs.coordTransMode == InterpolateCoordTransMode::pytorch_half_pixel && out_shape[2] > 1 && out_shape[3] > 1) ||
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:18]: I would still recommend to keep variables `index_h` and `index_w` for better code readability.

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:1]: Just a side note: do we really need to call `InterpolateExecutor::init(aclInterpolateAttrs, srcDescs, dstDescs, attr);` inside `ACLInterpolateExecutor::init`? Seems irrelevant to ACL executor

Comment (alvoron) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:1]: it seems it's not needed, deleted

**Review by dmitry-gorokhov — APPROVED** [review]

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp
Comment (alvoron) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:1]: Brought this line back since `InterpolateExecutor::init` did some initialization that requires ACL executor.
Without this line some tests segfaults.
